packages/lodestar/src/sync/utils/sync.ts

```diff
@@ -85,7 +86,8 @@ export function fetchBlockChunks(
   reqResp: IReqResp,
   getPeers: () => Promise<PeerId[]>,
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  maxBlocksPerChunk?: number
+  maxBlocksPerChunk?: number,
```

Comment:
I think you can remove this, we don't even use this in method

Reply:
Better to do it in another PR to not mixup changes?